package.json

```diff
@@ -1,10 +1,19 @@
 {
-  "version": "1.1.5",
-  "name": "mashape-analytics",
-  "description": "Node Agent for Mashape Analytics (https://analytics.mashape.com)",
+  "version": "2.0.0",
+  "name": "galileo-agent",
+  "description": "Node Agent for Galileo (https://getgalileo.io)",
   "author": "Mashape Inc. <opensource@mashape.com> (https://www.mashape.com/)",
-  "homepage": "https://analytics.mashape.com",
-  "repository": "Mashape/analytics-agent-node",
+  "contributors": [
```

Comment:
lets remove contributors entry altogether, keep the author as the only point of contact.